packages/base/src/ssr-dom-shim.ts

```diff
@@ -10,3 +10,56 @@ globalThis.Node ??= NodeShim as object as typeof Node;
 
 class FileListShim {}
 globalThis.FileList ??= FileListShim as object as typeof FileList;
+
+// ------- JS DOM shims -------
```

Comment:
@Lukas742 there were too many if statements in the code, so instead I started adding jsdom shims/polyfills to this file. It is loaded when running in node, so it is safe to add more jsdom specific polifylls/shims in the future here in case you see more failing scenarios in jest testing.

Reply:
Good to know, thanks for bringing this to my attention! 👍